caproto/asyncio/client.py

```diff
@@ -214,7 +214,12 @@ async def _broadcaster_receive_loop(self):
         'Loop which consumes receive_queue datagrams from _DatagramProtocol.'
         queues = collections.defaultdict(list)
         while True:
-            bytes_received, address = await self.receive_queue.async_get()
+            _, bytes_received, address = await self.receive_queue.async_get()
+            if isinstance(bytes_received, Exception):
```

Comment:
An exception in the middle of a tuple of three things seems a little surprising. Is it worth rethinking this, or would that require major changes?

Reply:
What would you suggest, the entire queue item being either an `Exception` or `tuple`?

Reply:
Can be refactored later, I suppose.